src/PrimeBooking.Domain/Informative/Hotel/Hotel.cs

```diff
@@ -0,0 +1,39 @@
+using PrimeBooking.Domain.Informative.Common;
+using PrimeBooking.Domain.Informative.Hotel.Errors;
+using PrimeBooking.Domain.Informative.Hotel.Events;
+
+namespace PrimeBooking.Domain.Informative.Hotel;
+
+public sealed class Hotel : AggregateRoot<HotelId>
+{
+    public string Name { get; private set; }
+    public int Capacity { get; private set; }
+    public ContactInformation ContactInformation { get; private set; }
+    public ICollection<Facility> Facilities { get; private set; }
+    public ICollection<Star> Stars { get; private set; }
+    
+    private Hotel()
+    {
+    }
+
+    public Hotel(string name, int capacity, ContactInformation contactInformation, ICollection<Facility> facilities, ICollection<Star> stars)
+    {
+        Name = name;
+        Capacity = capacity;
+        ContactInformation = contactInformation;
+        Facilities = facilities;
+        Stars = stars;
+    }
+
+    public Result Create()
+    {
+        if (string.IsNullOrEmpty(Name))
```

Comment:
We need to add here more validation cases. 
As it's an aggregate root, we must treat them in one place. 
Proposals, please add more:
1. Name is null or empty
2. Capacity > 0
3. ContactInformation, regex for Email and Phone.
4. Address: not sure what we need to validate here.

Reply:
Fixed